lib/grape.rb

```diff
@@ -133,6 +134,8 @@ module Presenters
   end
 end
 
+include Grape::Http::Headers
```

Comment:
Not be necessary.I just want to write `HTTP_ACCEPT_VERSION` instead of `Grape::Http::Headers::HTTP_ACCEPT_VERSION`, less words,but maybe it's not a good idea.